src/rgw/rgw_rados.h

```diff
@@ -749,9 +749,9 @@ class RGWSystemMetaObj {
   int use_default(bool old_format = false);
 
 public:
-  RGWSystemMetaObj() {}
-  RGWSystemMetaObj(const string& _name): name(_name)  {}
-  RGWSystemMetaObj(const string& _id, const string& _name) : id(_id), name(_name) {}
+  RGWSystemMetaObj() : cct(NULL), store(NULL) {}
```

Comment:
AFAIK initialization of members in the init list should be faster than in the constructor body because the compiler don't need to create a temporary object. I have check, but I would leave it for now as it is. If we change it we should change it everywhere.